src/android/io/branch/BranchSDK.java

```diff
@@ -259,6 +259,20 @@ private void initSession(CallbackContext callbackContext)
     }
 
     /**
+     * <p>Initialises a session with the Branch API, without a callback or {@link Activity}.</p>
+     */
+    private void initSession()
```

Comment:
@aaustin , Just have refactored the code, let me know if that is what you meant?